bundles/at.medevit.elexis.agenda.ui/rsc/html/switchParallel.html

```diff
@@ -142,18 +142,43 @@
 
       // https://fullcalendar.io/docs/v3/resourceOrder
       
+      var colorMapping = {};
       function loadResources(callback) {
         var _resources = loadResourcesFunction();
         var resources = JSON.parse(_resources);
+          resources.forEach(function(resource) {
+            colorMapping[resource.id] = 'rgba(' + resource.color + ', 0.2)';
+          });
         callback(resources);
-      }      
+      }
+
+      function getColorForResource(resourceId) {
+        var color = colorMapping[resourceId] || 'rgba(255, 255, 255, 0.2)';
+        return color;
+      }
 
       function loadEventsCallback(start, end, timezone, callback) {
         var eventsJson = loadEventsFunction(start.format(), end.format(), timezone);
         var jsonEvents = JSON.parse(eventsJson);
-
         var events = [];
-        jsonEvents.forEach(function (event) {
+          jsonEvents.forEach(function (event) {
+        var resourceId = event.resource || event.resourceId;
+        var color = getColorForResource(resourceId);
+        var autoStartDate = new Date();
+          autoStartDate.setFullYear(autoStartDate.getFullYear() - 1);
+        var autoEndDate = new Date();
+          autoEndDate.setFullYear(autoEndDate.getFullYear() + 3);
+        var formattedStart = autoStartDate.toISOString().split('T')[0] + "T00:00";
+        var formattedEnd = autoEndDate.toISOString().split('T')[0] + "T23:59";
+
+       	  events.push({
+            resourceId: resourceId,
+            start: formattedStart, 
+            end: formattedEnd, 
+            rendering: 'background',
+            color: color
+          });
+        

```

Comment:
Bitte hier noch den Code formattieren. So ist es schwirig zu sehen was in dem `jsonEvents.forEach(function (event) {` Block passiert.